src/modules/dashboard/components/ColonyHome/ColonyHome.tsx

```diff
@@ -220,21 +240,73 @@ const ColonyHome = ({
     />
   );
 
+  const renderNewTaskButton = (
+    <>
+      {canCreateTask && (
+        <ActionButton
+          button={({ onClick, disabled, loading }) => (
+            <Button
+              appearance={{ theme: 'primary', size: 'large' }}
+              text={MSG.newTaskButton}
+              disabled={disabled}
+              loading={loading}
+              onClick={throttle(onClick, 2000)}
+            />
+          )}
+          disabled={isInRecoveryMode}
+          error={ActionTypes.TASK_CREATE_ERROR}
+          submit={ActionTypes.TASK_CREATE}
+          success={ActionTypes.TASK_CREATE_SUCCESS}
+          transform={transform}
+          loading={isTaskBeingCreated}
+        />
+      )}
+    </>
+  );
+
+  const crumbs = (domains || [])
+    .sort((a, b) => a.id - b.id)
+    .reduce(
+      (accumulator, element) => {
+        if (element.id <= filteredDomainId) {
+          accumulator.push(element.name);
+        }
+        return accumulator;
+      },
+      ['Root'],
```

Comment:
There's two things here:
- This should be a message descriptor _(But this might be a little hard, please look into it)_
- As everywhere else, the root domain should be lowercase, ie: `root`